apps/pushes/management/commands/fork.py

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+'''Command to fix any of the repositories in the database
+that miss changeset mappings.
+'''
+from __future__ import absolute_import
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.six.moves import input
+
+from life.models import Forest, Repository
+
+
+class Command(BaseCommand):
+    help = 'Declare a Repository or Forest to be a fork '\
+        'to share the local clone'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--dry-run', '-n', action='store_true',
+                            help='Dry run, do not change db or repos')
+        parser.add_argument('orig',
+                            help='Repository or Forest to fork')
+        parser.add_argument('fork',
+                            help='Forked Repository or Forest')
+
+    def handle(self, orig, fork, dry_run=False, **options):
+        import hglib
```

Comment:
Legacy coding style. Back in the days, django was a just part of the repo, so you could run at least ./manage.py help without having python dependencies installed.

As django itself is now in the virtualenv, this is obsolete, I moved the import to the top-level.

Might make sense to do that globally, but that's a follow-up.